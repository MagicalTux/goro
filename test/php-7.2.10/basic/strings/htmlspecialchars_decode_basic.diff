     *** Testing htmlspecialchars_decode() : basic functionality ***
002- string(92) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
003- string(92) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
004- string(92) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
005- string(92) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
006- string(102) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. &quot; double quoted string &quot;"
007- string(102) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. &quot; double quoted string &quot;"
008- string(82) "Roy's height > Sam's height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
009- string(82) "Roy's height > Sam's height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
010- Done
002+ 
003+ Fatal error: Uncaught Error: Call to undefined function htmlspecialchars_decode in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/htmlspecialchars_decode_basic.php:16
004+ Stack trace:
005+ #0 {main}
006+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/htmlspecialchars_decode_basic.php on line 16
