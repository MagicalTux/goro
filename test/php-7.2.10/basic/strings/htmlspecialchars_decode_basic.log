
---- EXPECTED OUTPUT
*** Testing htmlspecialchars_decode() : basic functionality ***
string(92) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
string(92) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
string(92) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
string(92) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
string(102) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. &quot; double quoted string &quot;"
string(102) "Roy&#039;s height > Sam&#039;s height. 13 < 25. 1111 & 0000 = 0000. &quot; double quoted string &quot;"
string(82) "Roy's height > Sam's height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
string(82) "Roy's height > Sam's height. 13 < 25. 1111 & 0000 = 0000. " double quoted string ""
Done
---- ACTUAL OUTPUT
*** Testing htmlspecialchars_decode() : basic functionality ***

Fatal error: Uncaught Error: Call to undefined function htmlspecialchars_decode in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/htmlspecialchars_decode_basic.php:16
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/htmlspecialchars_decode_basic.php on line 16
---- FAILED
