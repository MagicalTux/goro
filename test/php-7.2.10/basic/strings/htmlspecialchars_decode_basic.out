*** Testing htmlspecialchars_decode() : basic functionality ***

Fatal error: Uncaught Error: Call to undefined function htmlspecialchars_decode in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/htmlspecialchars_decode_basic.php:16
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/htmlspecialchars_decode_basic.php on line 16